modules/javafx.graphics/src/main/native-glass/monocle/egl/eglBridge.c

```diff
@@ -88,3 +88,52 @@ JNIEXPORT jboolean JNICALL Java_com_sun_glass_ui_monocle_EGLAcceleratedScreen_nE
     return answer;
 }
 
+JNIEXPORT jlong JNICALL Java_com_sun_glass_ui_monocle_EGLScreen_nGetHandle
+(JNIEnv *env, jclass clazz, jint idx)  {
```

Comment:
Since the JNI parameters are unused, it may not matter, but since this is an instance method, the second argument is a `jobject` not a `jclass`. The same issue applies to the rest. I see that you just copied a preexisting issue, so you might or might not want to fix it, or maybe just file a follow-on cleanup issue if you like.

Reply:
I filed JDK-8263778 to track this. I will check the other methods as well as part of this issue.